### PR TITLE
MM-32809 Remove Toggle Dark Mode for Windows enable on Linux

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/desktop/pull/1494

Updated:
- User's Guide > Getting Started > Using Mattermost Desktop App > App Options > Dark Theme
   - Updated documentation to clarify that Dark Mode is available for Linux only while Windows and macOS respect the System Console Appearance setting

### CODE REVIEW

File: source/help/apps/desktop-guide.rst [hunk 1]
```diff
@@ -85,9 +85,13 @@ Leave app running in notification center when application window is closed (Linu
 Use GPU hardware acceleration
     When enabled, the Mattermost user interface is rendered more efficiently but can lead to decreased stability for some systems. This setting is enabled by default. You must restart the app to change this setting.
     
-Dark Theme
-~~~~~~~~~~
+Dark Theme (Linux only)
```
Comment (esethna) [inline source/help/apps/desktop-guide.rst:88]: The way this heading reads is that dark mode is available on linux only, but that's not the case. Mac and Windows are just based on your OS.

File: source/help/apps/desktop-guide.rst [hunk 2]
```diff
@@ -85,9 +85,13 @@ Leave app running in notification center when application window is closed (Linu
...
+~~~~~~~~~~~~~~~~~~~~~~~
 
-On macOS, the Mattermost desktop app respects the System Preferences appearance setting of the operating system to set the theme of the title bar. On Windows, you can toggle the theme of the Mattermost desktop app title bar in **... > View > Toggle Dark Mode**.  
+On Linux, you can run the Mattermost desktop app in dark mode by selecting Mattermost, then selecting **View > Toggle Dark Mode**.
 
 .. image:: ../../images/dark_theme.png
+
+On Windows and macOS, the Mattermost desktop app respects the operating system Appearance setting configured through System Preferences.
```
Comment (esethna) [inline source/help/apps/desktop-guide.rst:95]: Propose moving this above the linux blurb since win and mac are more common OSs

**Review by esethna — CHANGES_REQUESTED** [review]

**Review by esethna — APPROVED** [review]

File: source/help/apps/desktop-guide.rst (hunk 2 above)
Comment (devinbinnie) [reply source/help/apps/desktop-guide.rst:91]: We always only want to mention Windows 10, as Windows 7 doesn't have a dark mode. I don't think we support dark mode on Windows 7 currently though.

File: source/help/apps/desktop-guide.rst [hunk 3]
```diff
@@ -88,6 +88,8 @@ Use GPU hardware acceleration
 Dark Theme
 ~~~~~~~~~~
 
-On macOS, the Mattermost desktop app respects the System Preferences appearance setting of the operating system to set the theme of the title bar. On Windows, you can toggle the theme of the Mattermost desktop app title bar in **... > View > Toggle Dark Mode**.  
+On Windows and macOS, the Mattermost desktop app respects the operating system Appearance setting configured through System Preferences.
```
Comment (cwarnermm) [inline source/help/apps/desktop-guide.rst:91]:
```suggestion
On Windows 10 and macOS, the Mattermost desktop app respects the operating system Appearance setting configured through System Preferences.
```

File: source/help/apps/desktop-guide.rst (hunk 2 above)
Comment (cwarnermm) [reply source/help/apps/desktop-guide.rst]: Thanks, @devinbinnie! I've updated the text to reflect Windows 10.

**Review by devinbinnie — APPROVED** [review]

File: source/help/apps/desktop-guide.rst [hunk 4]
```diff
@@ -88,6 +88,8 @@ Use GPU hardware acceleration
 Dark Theme
 ~~~~~~~~~~
 
-On macOS, the Mattermost desktop app respects the System Preferences appearance setting of the operating system to set the theme of the title bar. On Windows, you can toggle the theme of the Mattermost desktop app title bar in **... > View > Toggle Dark Mode**.  
+On Windows 10 and macOS, the Mattermost desktop app respects the operating system Appearance setting configured through System Preferences.
```
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:91]:
```suggestion
On Windows 10 and macOS, the Mattermost Desktop App respects the operating system **Appearance** setting configured through System Preferences.
```

File: source/help/apps/desktop-guide.rst [hunk 5]
```diff
@@ -88,6 +88,8 @@ Use GPU hardware acceleration
...
+
+On Linux, you can run the Mattermost desktop app in dark mode by selecting Mattermost, then selecting **View > Toggle Dark Mode**.
```
Comment (justinegeffen) [inline source/help/apps/desktop-guide.rst:93]:
```suggestion
On Linux, you can run the Mattermost Desktop App in dark mode by selecting Mattermost, then selecting **View > Toggle Dark Mode**.
```

**Review by justinegeffen — COMMENTED** [review]
I *think* we're using Desktop App and not desktop app but I absolutely stand corrected on that!

**Comment by cwarnermm** [comment]
Thanks, @justinegeffen -- we tend to use Desktop App more frequently than other variations, so let's go with that! :)

**Review by justinegeffen — APPROVED** [review]

**Comment by amyblais** [comment]
@cwarnermm There are some merge conflicts here, can you help resolve and then we can merge this?

**Pull request merged**